source/OptimaJet.DWKit.StarterApplication/Services/HangfireActivator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+namespace Optimajet.DWKit.StarterApplication.Services
+{
+    public class HangfireActivator : Hangfire.JobActivator
```

Comment:
what's the purpose of this class? it only has a getter (but the getter is named like an action)

Reply:
It is there for Hangfire integration with AspNetCore DI.

Reply:
I'd add some documentation then, since it doesn't do anything.
(And I clearly missed it in the list of services being added).

Generally, if a class doesn't do anything, or has some unintuitive purpose, it needs at least a class header documentation block